tuning/sft_trainer.py

```diff
@@ -80,15 +81,15 @@ def _track_loss(self, loss_key, log_file, logs, state):
             return
 
         # append the current log to the jsonl file
-        with open(log_file, "a") as f:
+        with open(log_file, "a", encoding="utf-8") as f:
             f.write(f"{json.dumps(log_obj, sort_keys=True)}\n")
 
 
 def train(
     model_args: configs.ModelArguments,
     data_args: configs.DataArguments,
     train_args: configs.TrainingArguments,
-    peft_config: Optional[
+    peft_configs: Optional[
```

Comment:
Why did you update the name to `peft_configs`? Was this a requirement from pylinter? Only a single config should be passed in for training, either the LoraConfig or the PromptTuningConfig so it makes sense to leave it as singular

Reply:
@anhuong I was getting this lint error: `tuning/sft_trainer.py:91:4: W0621: Redefining name 'peft_config' from outer scope (line 26) (redefined-outer-name)`
I could revert this and mark it `# pylint: disable=redefined-outer-name`. What do you think?

Reply:
Good to know about the error, okay to leave it then